lib/rules/custom-property-no-missing-var-function/index.js

```diff
@@ -56,17 +54,19 @@ function rule(actual) {
 				return false;
 			});
 		});
-
-		function isKnownCustomProperty({ value }) {
-			return customProperties.has(value);
-		}
```

Comment:
[note] This change inlines `isKnownCustomProperty()` to avoid adding a `@param` type annotation.